TicTacToe/tictactoe.py

```diff
@@ -0,0 +1,51 @@
+list_of_winners_to_ensure_only_one_player_won = []
+
+def column_check(player, column_number):
+    global list_of_winners_to_ensure_only_one_player_won
```

Comment:
it will work without using global.
can you tell me why?

Reply:
Stackoverflow told me: The global statement can only be used at the very top of a function, and it is only necessary when you are assigning to the global variable. If you are just modifying a mutable object it does not need to be used.